### PR TITLE
Pascal.yaml: Adds Pascal schema

### PR DESCRIPTION
**Pull request by SerekKiri**
Pascal.yaml --> Added new language schema: Pascal

### CODE REVIEW

File: data/Language/Pascal.yaml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+identifier: Pascal
+creation_date: 1970
+wikidata: Q81571
+grammar_file: https://github.com/antlr/grammars-v4/blob/master/pascal/pascal.g4
+aliases:
+  - Pascal
+extensions:
```
Comment (ksdme) [inline data/Language/Pascal.yaml]: Wikipedia mentions `.inc` as a Pascal file extension.

File: data/Language/Pascal.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  - p
+  - pl
+line_continuation: '+'
+delimiters:
+  - double_slash
+  - curly_braces
+  - single_quote_slash_escape
+  - round_braces
+  - square_braces
+  - curly_braces
```
Comment (ksdme) [inline data/Language/Pascal.yaml]: Already mentioned this on line 15.

File: data/Language/Pascal.yaml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  - type
+  - unit
+  - until
+  - uses
+  - var
+  - while
+  - with
+  - xor
+datatypes:
+  - Integer
```
Comment (jayvdb) [inline data/Language/Pascal.yaml]: create datatype descriptions for at least the numerical datatypes.

also, doesnt Pascal have strings? :P

Comment (li-boxuan) [reply data/Language/Pascal.yaml]: Also boolean is missing :P

File: data/Language/Pascal.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  - var
+  - while
+  - with
+  - xor
+datatypes:
+  - String
+  - Boolean
+  - Integer
+  - real
+  - Character
```
Comment (li-boxuan) [inline data/Language/Pascal.yaml]: `char`, not `character`

File: data/Language/Pascal.yaml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  - or
+  - packed
+  - procedure
+  - program
+  - record
+  - repeat
+  - set
+  - shl
+  - shr
+  - string
```
Comment (li-boxuan) [inline data/Language/Pascal.yaml]: Remove this since it's already in `datatypes`

File: data/Language/Pascal.yaml [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+  - type
+  - unit
+  - until
+  - uses
+  - var
+  - while
+  - with
+  - xor
+datatypes:
+  - String
```
Comment (li-boxuan) [inline data/Language/Pascal.yaml]: Can you use lower case? Using the capital letter also passes compilation, but it is more common to use small letter. Also for other datatypes.

File: data/Language/Pascal.yaml (hunk 2 above)
Comment (li-boxuan) [reply data/Language/Pascal.yaml]: This is not addressed 😉

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

File: data/Language/Pascal.yaml [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+  - in
+  - inline
+  - interface
+  - label
+  - mod
+  - nil
+  - not
+  - object
+  - of
+  - on
```
Comment (jayvdb) [inline data/Language/Pascal.yaml]: truthy value is not quoted (truthy)

*Origin: YAMLLintBear, Section: `yaml`.*

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/30/commits/b8e643bc128c040245736da2b7d0b6b547648c8d

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**